soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -217,11 +217,25 @@ static ALWAYS_INLINE void clock_init(void)
 
 #if defined(CONFIG_I2C_MCUX_LPI2C) && \
 	(DT_NODE_HAS_STATUS(DT_NODELABEL(lpi2c1), okay) \
-	|| DT_NODE_HAS_STATUS(DT_NODELABEL(lpi2c2), okay))
+	|| DT_NODE_HAS_STATUS(DT_NODELABEL(lpi2c2), okay) \
```

Comment:
Ideally this should be split- so that we only clock LPI2C5/6 if the `lpi2c5` or `lpi2c6` dts nodes are enabled

Reply:
Updated.